packages/@aws-cdk/aws-apprunner-alpha/lib/service.ts

```diff
@@ -656,6 +657,18 @@ export interface ServiceProps {
    */
   readonly autoDeploymentsEnabled?: boolean;
 
+  /**
+   * Specifies an App Runner Auto Scaling Configuration.
+   *
+   * A default configuration is either the AWS recommended configuration,
+   * or the configuration you set as the default.
+   *
+   * @see https://docs.aws.amazon.com/apprunner/latest/dg/manage-autoscaling.html
+   *
+   * @default - use a default Auto Scaling Configuration.
```

Comment:
```suggestion
   * @default - the latest revision of a default auto scaling configuration is used.
```
[Docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apprunner-service.html#cfn-apprunner-service-autoscalingconfigurationarn)